pkg/command/plugin_bundle.go

```diff
@@ -51,6 +53,9 @@ to an internet-restricted environment. Please also see the "upload-bundle" comma
 			if !dpbo.dryRun && dpbo.tarFile == "" {
 				return errors.New("flag '--to-tar' is required")
 			}
+			if _, err := os.Stat(dpbo.tarFile); err == nil {
```

Comment:
I'm curious, why do the check here instead of in `verifyTarFile()` where we already do some validation?